app/routers/export.py

```diff
@@ -91,13 +124,30 @@ def add_attendees(ievent, attendees: list):
     return ievent
 
 
-def event_to_ical(user_event: UserEvent, attendees: List[str]) -> bytes:
+def event_to_ical(user_event: Event, attendees: List[str]) -> bytes:
     """Returns an ical event, given an
-    "UserEvent" instance and a list of email."""
+    "Event" instance and a list of email."""
 
     ical = create_ical_calendar()
     ievent = create_ical_event(user_event)
     ievent = add_attendees(ievent, attendees)
     ical.add_component(ievent)
 
     return ical.to_ical()
+
+
+def export_calendar(session: Session, events: List[Event]) -> bytes:
```

Comment:
This looks to my like an `internal` function :)

Reply:
if I will only move `export_calendar` into `internal` folder, it will create a circular import.
Shuold I move the other functions and keep just the `export `route?

Reply:
Why? The `internal` shouldn't use things from `routers`